cryptography/Cartfile

```diff
@@ -0,0 +1,2 @@
+github "wireapp/wire-ios-cryptobox" ~> 31.0.0
+github "AFNetworking/AFNetworking"
```

Comment:
Do we need `AFNetworking`?

Reply:
No, this was easier to verify compilation with